webpack/assets/javascripts/react_app/components/HostDetails/DetailsCard/index.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import {
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  Card,
+  CardHeader,
+  CardTitle,
+  CardBody,
+  CardActions,
+  ClipboardCopy,
+} from '@patternfly/react-core';
+import { UserIcon } from '@patternfly/react-icons';
+import { foremanUrl } from '../../../common/helpers';
+import PowerStatusDropdown from './PowerStatusDropdown';
+import { translate as __ } from '../../../common/I18n';
+
+const DetailsCard = ({
+  id: hostID,
+  ip,
+  mac,
+  uuid,
+  comment,
+  owner_id: ownerID,
+  owner_name: ownerName,
+  hostgroup_name: hostgroupName,
+  hostgroup_id: hostgroupID,
+}) => (
+  <Card isHoverable>
+    <CardHeader>
+      <CardTitle>{__('Details')}</CardTitle>
+      <CardActions>
+        <PowerStatusDropdown hostID={hostID} />
+      </CardActions>
+    </CardHeader>
+    <CardBody>
+      <DescriptionList
+        isInlineGrid
+        columnModifier={{
+          default: '2Col',
+        }}
+      >
+          
+        <DescriptionListGroup>
+          <DescriptionListTerm>{__('IP Address')}</DescriptionListTerm>
+          <DescriptionListDescription>
+            <ClipboardCopy variant="inline-compact">{ip}</ClipboardCopy>
+          </DescriptionListDescription>
+        </DescriptionListGroup>
+        <DescriptionListGroup>
+          <DescriptionListTerm>{__('Mac Address')}</DescriptionListTerm>
+          <DescriptionListDescription>
+            <ClipboardCopy variant="inline-compact">{mac}</ClipboardCopy>
+          </DescriptionListDescription>
+        </DescriptionListGroup>
+        <DescriptionListGroup>
+          <DescriptionListTerm>{__('Host Group')}</DescriptionListTerm>
+          <DescriptionListDescription>
+            <a href={foremanUrl(`/hostgroups/${hostgroupID}/edit`)}>
+              {hostgroupName}
+            </a>
+          </DescriptionListDescription>
+        </DescriptionListGroup>
+        <DescriptionListGroup>
+          <DescriptionListTerm>{__('Host Owner')}</DescriptionListTerm>
+          <DescriptionListDescription>
+            <a href={foremanUrl(`/users/${ownerID}/edit`)}>
+              <span>
+                <UserIcon /> {ownerName}
+              </span>
+            </a>
+          </DescriptionListDescription>
+        </DescriptionListGroup>
+      </DescriptionList>
+      <hr />
+      <DescriptionList
+        isInlineGrid
+        columnModifier={{
+          default: '1Col',
+        }}
+      >
+        <DescriptionListGroup>
+          <DescriptionListTerm>BIOS UUID</DescriptionListTerm>
```

Comment:
What's your reason for including this? I can honestly say I've never cared about this.

Reply:
I've taken it from the mockups for this page. the UUID here is being discussed at the moment, so I guess the properties list here would be changed.